.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,6 +17,12 @@ addresses #...
 - ...
 - Step 3, etc.
 
+[comment]: # (Specify if something in particular should be looked at, or ignored, during review)
+Review notes (optional):
+
+[comment]: # (Specify if something specific has to be tested, for example upgrade paths)
```

Comment:
Could we have those comment as md comments (similar to what has been done in `ideas` rep). This way I don't have to remove it :)